pyproject.toml

```diff
@@ -28,11 +28,11 @@ dependencies = [
     "fastapi>=0.112.0",
     "uvicorn",
     "requests",
-    "dls-dodal>=1.36.2",
-    "super-state-machine",  # https://github.com/DiamondLightSource/blueapi/issues/553
+    "dls-dodal>=1.43.0",
+    "super-state-machine",                           # https://github.com/DiamondLightSource/blueapi/issues/553
     "GitPython",
     "bluesky-stomp>=0.1.4",
-    "event-model==1.22.1",    # https://github.com/DiamondLightSource/blueapi/issues/684
+    "event-model==1.22.1",                           # https://github.com/DiamondLightSource/blueapi/issues/684
```

Comment:
We all seem to have different toml formatting settings in our vscodes, have noted in https://github.com/DiamondLightSource/python-copier-template/issues/207

In the meantime I should probably just go round the office when we're all in and check everyone's. Not the first time I've seen this.

Reply:
I don't have anything custom on mine, i just use whatever the repo gives me. Would you like me to change this back or leave it?

Reply:
Leaving is fine, we can sort it out properly later